case-lib/lib.sh

```diff
@@ -459,6 +459,32 @@ is_zephyr()
     test "$znum" -gt 10
 }
 
+is_IPC4()
+{
+    local ipc_type
+
+    ipcFile=/sys/module/snd_sof_pci/parameters/ipc_type
+    # If no /sys/module/snd_sof_pci/parameters/ipc_type exists
+    # the DUT is running IPC3 mode
+    [ -e "$ipcFile" ] || {
+        dlogi 'No /sys/module/snd_sof_pci/parameters/ipc_type, DUT runs IPC3 mode'
+        return 1
+    }
+
+    # If /sys/module/snd_sof_pci/parameters/ipc_type exists
+    # If the value of file ipc_type is:
+    # 0: DUT runs IPC3 mode
+    # 1: DUT runs IPC4 mode
+    ipc_type=$(cat $ipcFile)
+    if [ $ipc_type -eq 1 ]; then
+        dlogi "/sys/module/snd_sof_pci/parameters/ipc_type is ${ipc_type}, DUT runs IPC4 mode"
```

Comment:
Sorry, by "runs" does it mean "will run in?"  Or does it mean it ran in IPC4 mode?  Current tense is ambiguous here and below.

Reply:
hi @greg-intel  I modified the tense in #884 .
@plbossart Sorry, I mistook the repo . Already prepare a new PR from my own forked repo #884  , thx~